src/Analyzer/FileVisitor.php

```diff
@@ -41,17 +41,29 @@ public function enterNode(Node $node): void
          * @see FileVisitorTest::test_should_returns_all_dependencies
          */
         if ($node instanceof Node\Expr\StaticCall && method_exists($node->class, 'toString')) {
```

Comment:
Could this condition be expressed implementing a specific method in Node\Expr\StaticCall?

Reply:
I think not because is there is no method toString it can generate an exception

Reply:
Hmm, maybe I read better now... so is $node->class a class outside of phparkitect?
Is it a class of the project that phparkitect is checking out?